src/nightwatch/snapshot.js

```diff
@@ -0,0 +1,37 @@
+const Jimp = require('jimp');
+const fs = require('fs');
+const path = require('path');
+
+
+module.exports = function (browser, done) {
+  const testDirectory = path.dirname(module.parent.filename);
+  const currentTest = browser.currentTest;
+
+  browser.execute('document.documentElement.getAttribute("lang")', [], (locale) => {
+    const snapshotDirectory = path.join(testDirectory, '__snapshots__', locale.value || 'en');
```

Comment:
Probably be good to fallback to en-US since that is the site's starting locale

Reply:
👍 was also thinking of allowing this to be configurable in nightwatch config